src/popup/components/ToggleMultisigButton.vue

```diff
@@ -4,23 +4,31 @@
     class="multisig-button"
     @click="toggleMultisigDashboard(!isMultisig)"
   >
-    <span
-      v-if="isMultisig"
-      class="text"
-    >
-      {{ $t('multisig.backToMainAccounts') }}
-    </span>
+    <template v-if="isMultisig">
+      <ArrowBackIcon
+        class="icon"
+      />
+      <span class="text">
+        {{ $t('multisig.backToMainAccounts') }}
+      </span>
+    </template>
+
     <template v-else>
       <div
         v-if="hasPendingMultisigTransaction"
-        class="icon-wrapper"
+        class="has-pending"
       >
         <PendingIcon
-          class="animated-pending-icon"
+          class="icon"
```

Comment:
I think it's a bad change to shorten the class name. Those two icons used in this component has different styles. Maybe you can share the common part this way:

```
.icon-foo,
.icon-bar {
  // Common styles
}
```